backend/migrations/0002_message_created_at.up.sql

```diff
@@ -0,0 +1,2 @@
+-- Add up migration script here
+ALTER TABLE message ADD COLUMN created_at TIMESTAMPTZ NOT NULL DEFAULT NOW();
```

Comment:
xxT1m3StamPzZ

Reply:
lol.  Yes `timestampz` is a time-zone aware timestamp in postgresql 
https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-timestamp/